database/migrations/2024_08_24_213634_create_semesters_table.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class CreateSemestersTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('semesters', function (Blueprint $table) {
+            $table->id();
+            $table->string('title', 255);
+            $table->integer('version');
+            $table->enum('type', ['Undergraduate', 'Postgraduate']);
+            $table->text('description')->nullable();
+            $table->string('url', 255)->nullable();
+            $table->timestamps(); // This will create `created_at` and `updated_at` fields automatically
+            $table->foreignId('created_by')->constrained('users'); 
+            $table->foreignId('updated_by')->constrained('users');  
+        });
```

Comment:
Model definition have something else. Can you re-check? 

```
'created_at' => 'datetime',
'updated_at' => 'datetime',
```

Reply:
Do we need define 'created_at' and 'updated_at' manually because I have added $table->timestamps(); It creates created_at and updated_at columns automatically